### PR TITLE
Major upgrades to utilities, DSN now implemented in pure python

### PR DESCRIPTION
**Pull request by dPys**

### CODE REVIEW

File: ndmg/utils/gen_utils.py [hunk 1]
```diff
@@ -42,6 +41,20 @@
...
+    template_img_data = nib.load(template_path).get_data().astype('bool')
+    template_actor = actor.contour_from_roi(template_img_data,
+                                            color=(50, 50, 50), opacity=0.05)
+    renderer.add(template_actor)
+    lines_actor = actor.streamtube(final_streamlines, window.colors.orange,
+                                   linewidth=0.3)
+    renderer.add(lines_actor)
+    window.record(renderer, n_frames=1, out_path=fname, size=(900, 900))
+    return
+
```
Comment (loftusa) [inline ndmg/utils/gen_utils.py:57]: pep-8 doesn't generally recommend imports inside of functions: https://www.python.org/dev/peps/pep-0008/#imports
doesn't matter too much in this case, but just FYI

File: setup.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
         "ndmg.nuis",
         "ndmg.scripts",
     ],
+    include_package_data = True,
```
Comment (loftusa) [inline setup.py:31]: not sure what this does

**Review by loftusa — COMMENTED** [review]
didn't understand most of the changes to gen_reg, but looks good
Gonna test it myself before approving, one sec

File: ndmg/utils/gen_utils.py (hunk 1 above)
Comment (dPys) [reply ndmg/utils/gen_utils.py:57]: For sure, I'm used to this from working with nipype where function nodes have their own name spaces!

**Pull request merged**